LayoutTests/http/tests/site-isolation/resources/getDisplayMedia-starts-frame.html

```diff
@@ -0,0 +1,10 @@
+<script>
+window.internals.settings.setScreenCaptureEnabled(true);
```

Comment:
This setting doesn't look like it exists.

Reply:
`ScreenCaptureEnabled` is in UnifiedWebPreferences and is also part of the built `JSInternalSettingsGenerated.cpp`?